widgets/more_buttons.py

```diff
@@ -0,0 +1,40 @@
+from PyQt5.QtWidgets import QWidget, QVBoxLayout, QPushButton
+from lib.libbol import *
+from widgets.tree_view import BolHeader
+
+class MoreButtons(QWidget):
+    def __init__(self, parent):
+        super().__init__(parent)
+        self.parent = parent
+        self.vbox = QVBoxLayout(self)
```

Comment:
At the moment, these buttons are slightly smaller than the others. Removing the margin solves it:

```python
self.vbox.setContentsMargins(0, 0, 0, 0)
```